src/views/CardStack.js

```diff
@@ -81,7 +81,8 @@ const RESPOND_THRESHOLD = 12;
 /**
  * The distance of touch start from the edge of the screen where the gesture will be recognized
  */
-const GESTURE_RESPONSE_DISTANCE = 35;
+const GESTURE_RESPONSE_DISTANCE_SIDE = 35;
```

Comment:
Thought top and side might be a bit more intuitive. Vertical and horizontal do fit with the code that's already there though. Change made